packages/flutter/test/material/input_decorator_test.dart

```diff
@@ -993,6 +993,45 @@ void main() {
     expect(tester.getTopRight(find.text('text')).dx, lessThanOrEqualTo(tester.getTopRight(find.byKey(sKey)).dx));
   });
 
+  testWidgets('InputDecorator tall prefix', (WidgetTester tester) async {
+    const Key pKey = Key('p');
+    await tester.pumpWidget(
+      buildInputDecorator(
+        // isEmpty: false (default)
+        // isFocused: false (default)
+        decoration: InputDecoration(
+          prefix: Container(
+            key: pKey,
+            height: 100,
+            width: 10,
+          ),
+          filled: true,
+        ),
+      ),
+    );
+
+    // Overall height for this InputDecorator is ~127.2dps because
```

Comment:
Assuming that the [Ahem font ](https://web-platform-tests.org/writing-tests/ahem.html)the tests are using is 16dps high then the baseline should be exactly 12.8 from the top. It would be helpful to understand why the epsilons are needed here. Are we doing too much arithmetic at layout time?

Reply:
Moved to 20dps font size and everything worked out as integers.

Reply:
Just a note for posterity, the Mac tests ended up having rounding errors even with the font size change.  I left the size at 20dps but added `closeTo` where needed.